music/models.py

```diff
@@ -1,3 +1,46 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+
+class Track(models.Model):
+    track_name = models.CharField(max_length=50)
+    artist = models.ForeignKey('Artist', on_delete=models.CASCADE, related_name='track')
+    album = models.ForeignKey('Album', on_delete=models.CASCADE, related_name='track')
+    genre = models.ManyToManyField('Genre', related_name='track')
+    length = models.CharField(max_length=10)
+    thumbnail = models.CharField(max_length=50)
+    song = models.FileField(upload_to='mp3')
+
+    def __str__(self):
+        return f"{self.track_name} {self.artist} {self.album}"
+
+
+class Artist(models.Model):
+    artist_name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.artist_name}"
+
+
+class Album(models.Model):
+    album_name = models.CharField(max_length=50)
+    artist = models.ForeignKey('Artist', on_delete=models.CASCADE, related_name='album')
+
+    def __str__(self):
+        return f"{self.album_name}"
+
+
+class Genre(models.Model):
+    genre_name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.genre_name}"
+
+
+class Playlist(AbstractUser):
+    playlist_name = models.CharField(max_length=50)
+    track = models.ManyToManyField('Track', related_name= 'playlist')
+    liked_songs = models.CharField(max_length=300, null=True)
```

Comment:
Remove this field and create a Playlist with a name of "Liked Songs" for every user that gets created

Reply:
'user.User'